pkg/spiceerrors/bug.go

```diff
@@ -0,0 +1,31 @@
+package spiceerrors
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// Based on: https://stackoverflow.com/a/58945030
+func isInTests() bool {
+	for _, arg := range os.Args {
+		if strings.HasPrefix(arg, "-test.v=") {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Let's add a test `bug_test.go` file that checks that `isInTests` is returning true while in test, in case there is any regression with newer go versions